docs/operators/key_value_parser.md

```diff
@@ -0,0 +1,178 @@
+## `key_value_parser` operator
+
+The `key_value_parser` operator parses the string-type field selected by `parse_from` into key value pairs. All values are of type string.
+
+### Configuration Fields
+
+| Field            | Default             | Description                                                                                                                                                                                                                              |
```

Comment:
Nit: To be consistent with other docs, can you add punctuation to the descriptions.